scripts/ngless-count.py

```diff
@@ -20,80 +21,48 @@ def parse_args():
                         help="SAM/BAM/CRAM file to count reads on")
     parser.add_argument("-o", "--output", required=True,
                         help="Output file/path for results")
+    parser.add_argument("-f", "--features",
+                        choices=["seqname", "gene"],
```

Comment:
Need to check if there's an option to allow unlisted choices. My initial plan was documenting `--help` and show choices as part of the errors (also picked up by the cwl tools).